providers/googlemaps/googlemaps.go

```diff
@@ -0,0 +1,25 @@
+package googlemaps
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/zquestz/s/providers"
+)
+
+func init() {
+	providers.AddProvider("googlemaps", &Provider{})
+}
+
+// Provider merely implements the Provider interface.
+type Provider struct{}
+
+// BuildURI generates a search URL for Google Maps.
+func (p *Provider) BuildURI(q string) string {
+	return fmt.Sprintf("https://www.google.com/maps/search/?api=1&query=%s", url.QueryEscape(q))
+}
+
+// Tags returns the tags relevant to this provider.
+func (p *Provider) Tags() []string {
+	return []string{"search"}
```

Comment:
I think this should be left blank, search really indicates a more general purpose search like google/duckduckgo/etc. Most users won't want to search maps as well.

Reply:
Reminder, we should remove this tag. =)